test/include/autoinstall/AutoinstallAskInclude_test.rb

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/env rspec
+
+root_path = File.expand_path('../..', __FILE__)
+ENV["Y2DIR"] = File.join(root_path, 'src')
+
+require "yast"
+require_relative "../../../src/include/autoinstall/ask.rb"
+require_relative "../../../src/modules/Profile"
+
+describe "Yast::AutoinstallAskInclude" do
+  module DummyYast
+    class AutoinstallAskClient < Yast::Client
+      def main
+        Yast.include self, "autoinstall/ask.rb"
+      end
+
+      def initialize
+        main
+      end
+    end
+  end
+
+  let(:client) { DummyYast::AutoinstallAskClient.new }
+  let(:base_ask) do
+    { "type" => "string", "question" => "hostname?", "default" => "my.site.de",
+      "help" => "Some help", "dialog" => 0, "element" => 0,
+      "stage" => "initial" }
+  end
+  let(:profile) { { "general" => { "ask-list" => ask_list } } }
+  let(:result) { :ok }
```

Comment:
OK, I am not much fan of having result in let as it decrease readability of expect states with more abstractions, but no strong agains, so just idea.
Idea is mainly that for tests it is better to be more specific and less abstract to prevent bugs in tests and allow easier read what is really expected output.